c-log/README.md

```diff
@@ -0,0 +1,188 @@
+# clog
```

Comment:
`Logger` is only defined in the standard library under the `log` package.  I see no problem having a `logger` package inside kayvee-go